### DISCUSSION TITLE
Facets and filter question

### DISCUSSION THREAD

**Discussion by kizzbonez** [bigcommerce/catalyst#1472] [Q&A] [answered]
Can I use  BC app like freshclick as an altenative for BC facets and filtering for catalyst?

**Comment by bookernath** [comment, marked as answer]
You can use any search solution that exposes APIs; we have a pre-built integration with Algolia for example: https://www.catalyst.dev/integrations/algolia

I'm not aware of FreshClick having updated their app to support Catalyst and in a quick check of their [docs](https://www.freshclick.co.uk/docs/product-filter-search.html) I don't see any public APIs that Catalyst could consume.

I'd recommend reaching out to FreshClick to inquire about their plans to support Catalyst; we are happy to support them in integrating.
